cmd/cloudServerCreate.go

```diff
@@ -134,17 +134,17 @@ func init() {
 		sshPubKeyFile = fmt.Sprintf("%s/.ssh/id_rsa.pub", home)
 	}
 
-	cloudServerCreateCmd.Flags().String("template", "", "template to use (see 'cloud server options --templates')")
+	cloudServerCreateCmd.Flags().String("template", cast.ToString(defaultFlag("template")), "template to use (see 'cloud server options --templates')")
 	cloudServerCreateCmd.Flags().String("type", "SS.VPS", "some examples of types; SS.VPS, SS.VPS.WIN, SS.VM, SS.VM.WIN")
 	cloudServerCreateCmd.Flags().String("hostname", "", "hostname to set")
 	cloudServerCreateCmd.Flags().Int("ips", 1, "amount of IP addresses")
 	cloudServerCreateCmd.Flags().String("public-ssh-key", sshPubKeyFile,
 		"path to file containing the public ssh key you wish to be on the new Cloud Server")
-	cloudServerCreateCmd.Flags().Int("config-id", 0, "config-id to use")
+	cloudServerCreateCmd.Flags().Int("config-id", cast.ToInt(defaultFlag("config-id", -1)), "config-id to use")
```

Comment:
I'm not sure if the default values make sense to be global.... for example,  a `config-id` default that is the same for create and resize and clone doesn't make sense to me.... I wonder if it should be something more specific... like `cloud.server.create.config-id`...

Reply:
:thinking: i'll give it some thought.

Reply:
@dwburke pushed up changes so its now specific to a command.